modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java

```diff
@@ -113,6 +113,8 @@ public void initContainer(ReadableByteChannel readableByteChannel, long containe
                 } else {
                     throw e;
                 }
+            } catch (RuntimeException e) {
```

Comment:
I don't like ignoring such a huge range of errors. Isn't there any more specific error that we can catch? Even better, is there any place in portal where we can catch the error and handle it properly?

Thanks!

Reply:
Hi @adolfopa if the error is caught in the portal, the temporary file will not be closed, and the issue will persist. I can maybe add logging in the catch block or extract the cause from the RuntimeException and only returning when there is a specific case. Please let me know if you like these ideas. Thanks!

Reply:
Yes, if there's no other choice, if we could at least log the error I'll be fine with the fix.

Reply:
OK, I added a logging statement.
https://github.com/liferay-lima/liferay-portal/pull/985